### DISCUSSION TITLE
请问【群成员增加事件】中能够直接获取到群名称吗？

### DISCUSSION THREAD

**Discussion by mark9804** [takayama-lily/oicq#312] [Q&A] [answered]
如题，我想通过群名称实现发送不同欢迎消息的功能，但是入群事件中原本应该含有 `group_name` 等信息的 `Group` 对象是空 Object，不包含群名称信息。

我也尝试了使用 `pickGroup(群号)` 来获取群名称，但是也返回了空对象：
```js
<< console.log("新增加群信息：" + JSON.stringify(msg.group));
>> 新增加群信息：{}

<< console.log(bot.pickGroup(msg.group_id));
>> Group {}
```

想请教一下有没有直接获取到群员增加事件中的 `group_name` 字段的方式，谢谢！

<img width="777" alt="image" src="https://user-images.githubusercontent.com/9006264/155257848-27991fe3-9b4e-41d6-b48e-3d270f0ebf9e.png">

**Comment by takayama-lily** [comment, marked as answer]
不是空的哦
https://github.com/takayama-lily/oicq#class-group

**Reply by mark9804** [reply]
感谢解答，之前一直在用 `group_name` ，刚刚卸载了星际才发现应该访问 `name` ……

**Reply by wuxianggujun** [reply]
> 感谢解答，之前一直在用 `group_name` ，刚刚卸载了星际才发现应该访问 `name` ……

我看了你的，应该怎么获取一个群对象

**Reply by mark9804** [reply]
> 应该怎么获取一个群对象

`pickGroup(群号)`

**Reply by wuxianggujun** [reply]
> > 应该怎么获取一个群对象
> 
> `pickGroup(群号)`

我也试过，是null的
